composer.json

```diff
@@ -21,15 +21,15 @@
         }
     ],
     "require": {
-        "silverstripe/admin": "^1@dev",
-        "silverstripe/framework": "^4@dev",
-        "silverstripe/versioned": "^1@dev",
+        "silverstripe/admin": "^1.5",
```

Comment:
```suggestion
        "php": ">=7.1",
        "silverstripe/admin": "^1.5",
```

Reply:
I don't agree with introducing this - we already list PHP requirements in framework, that's a single place where we need to maintain it at the moment

Reply:
Unfortunately that approach doesn't always work, it is what led to the original creation of this issue https://github.com/silverstripe/silverstripe-cms/issues/2521

What happened is that a travis build failed because there was a module (I forget which one) that requires silverstripe/cms but not silverstripe/framework.  That particular travis test was running php 5.6 and failed because it included a version of silverstripe/cms that required php 7.1

Reply:
If you're adding the PHP constraint then you don't need to bump the admin constraint

Reply:
Apologies, I may have misinterpreted the solution.  My suggestion for adding the PHP constraint is not required as the lock-stepping takes care of the original issue.